api/resources_portal/management/commands/material_additional_metadata_key_change.py

```diff
@@ -0,0 +1,119 @@
+from django.core.management.base import BaseCommand
+from django.db import transaction
+
+from resources_portal.config.logging import get_and_configure_logger
+from resources_portal.models import Material
+
+logger = get_and_configure_logger(__name__)
+
+
+def validateCategory(category):
+    return len([choice for choice in Material.CATEGORY_CHOICES if category in choice]) > 0
+
+
+def addKey(category, new_key):
+    querySet = None
+    materials_changed = 0
+
+    if category:
+        if not validateCategory(category):
+            logger.error(f"The provided category '{category}' is not a valid material category.")
+            return
+        querySet = Material.objects.filter(category=category)
+    else:
+        querySet = Material.objects.all()
+
+    with transaction.atomic():
+        for material in querySet:
+            material.additional_metadata[new_key] = ""
```

Comment:
It seems like the default value should be a parameter right?

Reply:
Just added it!